src/store/getters.js

```diff
@@ -1,15 +1,52 @@
+import { mnemonicToSeed } from '@aeternity/bip39';
 import BigNumber from 'bignumber.js';
 import { toShiftedBigNumber } from '@/utils';
 import {
+  ACCOUNT_HD_WALLET,
+  PROTOCOL_AETERNITY,
   TX_DIRECTION,
 } from '@/constants';
+import { ProtocolAdapterFactory } from '@/lib/ProtocolAdapterFactory';
 import { AE_SYMBOL } from '@/protocols/aeternity/config';
 import {
   aettosToAe,
   categorizeContractCallTxObject,
 } from '@/protocols/aeternity/helpers';
 
 export default {
+  wallet({ mnemonic }) {
+    if (!mnemonic) return null;
+    return mnemonicToSeed(mnemonic);
+  },
+  accounts({ accounts: { list } }, getters) {
+    if (!getters.wallet) {
+      return [];
+    }
+
+    return list
+      .map(({
+        idx, type, protocol = PROTOCOL_AETERNITY, ...acc
+      }, index) => ({
+        globalIdx: index,
+        idx,
+        type,
+        protocol,
+        ...acc,
+        ...(type === ACCOUNT_HD_WALLET
+          ? ProtocolAdapterFactory
+            .getAdapter(protocol)
+            .getHdWalletAccountFromMnemonicSeed(
+              getters.wallet,
+              idx,
+            )
+          : {}
+        ),
+      }))
+      .map(({ ...account }) => ({
+        ...account,
+        name: getters['names/getDefault'](account.address),
+      }));
+  },
   getTxSymbol: ({ fungibleTokens: { availableTokens } }) => (transaction) => {
```

Comment:
This can be removed because you created same method within the `fungibleTokens` composable.